lib/checker.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+const readFileSync = require('fs').readFileSync;
+const stripJsonComments = require('strip-json-comments');
+const stylelintConfigPrettier = require("./index");
+
+exports.check = function (configFilePath) {
+	if (typeof configFilePath !== "string") {
+		throw new Error("Usage: stylelint-config-prettier-check <pathToConfigFile>");
+	}
+
+	const rawData = readFileSync(configFilePath, 'utf8', (err) => {
+		if (err) throw new Error(err);
+	});
+
+	const stylelintConfig = JSON.parse(stripJsonComments(rawData));
```

Comment:
Not all stylelint config files are JSON. Some are JS (including `./lib/index.js`), others are .stylelintrc files, and others are embedded into package.json (my personal favorite). How would you feel about [using `stylelint` itself to load the configuration](https://github.com/stylelint/stylelint/blob/f6fbad37e1ffa84aed4a996b3829b4275e7320c5/lib/__tests__/createLinter.test.js#L8-L15)? That would be closer to how `eslint-config-prettier` implements their checker.